assets/src/components/propertiesPanel/minischedule.tsx

```diff
@@ -323,10 +323,13 @@ const Piece = ({
   pieceIndex: number
   activeIndex: [number, number] | null
 }) => {
+  const startsWithMidRouteTrip = piece.startMidRoute ? true : false
   const isSwingOn: boolean =
     piece.trips.length > 0 &&
     isTrip(piece.trips[0]) &&
-    !isDeadhead(piece.trips[0])
+    !isDeadhead(
+      startsWithMidRouteTrip ? piece.startMidRoute!.trip : piece.trips[0]
+    )
```

Comment:
this is causing a warning about 
```
  331:32  warning  Forbidden non-null assertion  @typescript-eslint/no-non-null-assertion
```
So I'm considering reverting the change and going back to `piece.startMidRoute ? <a> : <b>`. 

That okay with you @KaylaBrady ?

Reply:
addressed in [mbta/skate@`52985e1` (#1843)](https://github.com/mbta/skate/pull/1843/commits/52985e175b13f54c74c74b33fb1bd7f3edd2e87b)